benchmarks/elementary_functions_experiments_benchmark.cpp

```diff
@@ -152,7 +152,9 @@ class SingleTableImplementation {
     Value cos_x;
   };
 
-  static constexpr Value cos_polynomial_0 = -0.499999999999999999999872434553;
+  // If this was ever changed to a value that is not a power of 2, extra care
+  // would be needed in the computations that use it.
```

Comment:
For that reason, I would much prefer inlining the -0.5 where this is used, since they need to change anyway if this changes.
This will also make it easier to review the error analyses etc.

Reply:
As discussed, it's useful to give a name to that number and make sure that it has the same value in the two places where it's used.